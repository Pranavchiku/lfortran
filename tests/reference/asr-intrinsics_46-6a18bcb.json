{
    "basename": "asr-intrinsics_46-6a18bcb",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_46.f90",
    "infile_hash": "ac53f7df73e923aad9bd851b7737582595700317ef702c3b5936bd38",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-intrinsics_46-6a18bcb.stdout",
    "stdout_hash": "e4ed1f54628c27edcc56bb4ce545e7ca8a20bda465f3ff476ae93c2e",
    "stderr": "asr-intrinsics_46-6a18bcb.stderr",
    "stderr_hash": "55969d43b3e0da1e6020f1c9bbed6aee836e2f17754ee64ebd0c8e97",
    "returncode": 0
}