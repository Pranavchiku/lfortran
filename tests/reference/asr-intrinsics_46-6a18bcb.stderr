warning: Implied save feature
 --> tests/../integration_tests/intrinsics_46.f90:7:5
  |
7 |     character(len=1) :: c_1 = 'b'
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
